utils/doclint/api_parser.js

```diff
@@ -118,20 +118,27 @@ class ApiParser {
    * @param {MarkdownNode} spec
    */
   parseArgument(spec) {
+    const langMatches =function (memberLangs, argumentLangs) {
```

Comment:
@pavelfeldman maybe I'm doing something dumb here. But I had to add this piece of code because my `serializerOptions` param was getting added to the `js, python` version of `postDataJSON` of the method. I need it in the csharp method version.

That reminded me that I use a different casing to fool that part.

thoughts?

Reply:
It works for me w/o your modifications :/